frontend/src/state/days.js

```diff
@@ -3,12 +3,16 @@ import { readable } from 'svelte/store';
 
 export default readable([], (set) => {
     const now = moment()
-    const then = now.clone().add(7, 'days')
-
     const days = []
-    while (now < then) {
-        days.push(now.format('YYYY-MM-DD'))
-        now.add(1, 'days')
+
+    while(days.length < 7) {
+        let day = now.day();
+
+        if (!(day == 6 || day == 0)) {
```

Comment:
not sure if we should actually prevent people from booking rooms on the weekend?

Reply:
I think it makes sense. I know of people who just reserved the weekend because they did not bother to check whether it actually was a weekday or not. They just booked it but never had the intention of actually being there.

Reply:
I'm fine though with not showing the weekend